node/src/actors/session/actor.rs

```diff
@@ -153,7 +153,7 @@ impl Session {
                     }
                     Ok(Err(CheckpointZeroInTheFuture(zero))) => {
                         let date = pretty_print(zero, 0);
-                        log::warn!("Checkpoint zero is in the future ({:?}). Delaying chain bootstrapping until then.", date);
+                        log::warn!("Network bootstrapping is scheduled for {:?}. The node will remain idle and delay chain bootstrapping until then. Wait for it!", date);
```

Comment:
This line is duplicated in `node/src/actors/chain_manager/actor.rs`, not sure why.